javascript/javascript2/week2/warm_up.js

```diff
@@ -0,0 +1,74 @@
+// Doubling of number
+let numbers = [1, 2, 3, 4, 5, 6];
+let newNumbers = [];
+
+newNumbers = numbers
+  .filter(element => element % 2 !== 0)
+  .map(element => (newNumbers = element * 2));
+console.log(`The double numbers are ${newNumbers}`);
+
+// Working with movies
+// I am using same sample that I have create for my own becuase the one that provided is quite big 
+const movieName = [
+  { title: "The three men and the pistol", year: 1980, rating: 7 },
+  { title: "Chase three - The final chase", year: 1981, rating: 5 },
+  { title: "Surfer", year: 1982, rating: 6 },
+  { title: "Alien", year: 2005, rating: 3 },
+  { title: "The Benjamin", year: 2014, rating: 8 },
+  { title: "Boat", year: 1979, rating: 9 },
+];
+
+// (1)
+const shortTitle = movieName.filter(element => element.title.split(" ").length <= 3);
+console.log(shortTitle)
+
+// (2)
+const longTitle = movieName.filter(element => element.title.split(" ").length > 3);
+console.log(longTitle)
+
+// (3)
+const movieYearMade = movieName.filter(element => element.year >= 1980 && element.year <= 1989);
```

Comment:
To count the number of movies made between 1980 and 1989 you could use the length method over the filtered array.
Example: 
```
const moviesBetween1980and1989 = movies.filter(
  (movie) => movie.year >= 1980 && movie.year <= 1989
).length
```